core-war/src/main/resources/repository.properties

```diff
@@ -0,0 +1,10 @@
+# The factory to build a CMIS service instance for the OpenCMIS framework
```

Comment:
*repository.properties* is a bit general naming, maybe *cmis-repository.properties* or *default-cmis-repository.properties* makes easier the linking with the CMIS feature?

Reply:
Done

Reply:
Finally I rollback the properties file name to repository.properties because this name is what is expected by the OpenCMIS framework